org.prismlauncher.PrismLauncher.yml

```
@@ -111,8 +112,8 @@ modules:
     sources:
       - type: archive
         archive-type: tar-gzip
```

```suggestion
        dest-filename: gamemode.tar.gz
```

And then next pull requests will be named "Update gamemode.tar.gz to 1.9", instead of how this one is named.